src/test/database/remove-remote.spec.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import { SinonSandbox } from "sinon";
```

Comment:
this import can be removed and the definition below of `sandbox` can be

`let sandbox: sinon.SinonSandbox`